Controller/CancelController.php

```diff
@@ -2,11 +2,16 @@
 namespace Payum\Bundle\PayumBundle\Controller;
 
 use Payum\Core\Request\Cancel;
+use Symfony\Component\HttpFoundation\RedirectResponse;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
 
 class CancelController extends PayumController
 {
+    /**
+     * @return RedirectResponse|Response
```

Comment:
The method can get a `Response` as the return type, since `RedirectResponse` extends from `Response`

Reply:
thanks. i changed it